INSTALL.md

```diff
@@ -841,6 +841,18 @@ Don't perform FIPS module Power On Self Tests.
 This option MUST be used for debugging only as it makes the FIPS provider
 non-compliant. It is useful when setting breakpoints in FIPS algorithms.
 
+### enable-fips-jitter
+
+Use the CPU Jitter library as a FIPS validated entropy source.
+
+This option will only produce a compliant FIPS provider if you have:
+
+1. independently performed the required [SP 800-90B] entropy assessments;
+2. met the minimum required entropy as specified by [jitterentropy-library] and
+3. have had the resulting module FIPS validated by the [CMVP].
+
+Failure to do all of these will produce a non-compliant FIPS provider.
```

Comment:
... And re-validate on every change/submission due to chaining. If on the other hand it is unchained like in https://github.com/openssl/openssl/pull/25275 then an existing certificate could be reused. Standalone static library has been validated before https://csrc.nist.gov/projects/cryptographic-module-validation-program/entropy-validations/certificate/191 that could be marked for reuse, with public downloads available.

Although self-submission of jitterentropy-library is straight forward.

Reply:
As noted above, chaining is not happening inside the FIPS boundary.

Reply:
I don't think certificate reuse is possible when changing platforms.  The quality of the entropy is strongly tied to the CPU and it's environment.  Any change there is going to throw the computation off and require a re-assessment.

Some changes might be hideable from NIST (e.g. upgrading main memory or changing a disc drive), but the reality is everything will impact the jitter entropy quality to some extent.

Reply:
Please mark as resolved